### PR TITLE
feat: extend kind list, kind types, kind translaions

### PR DESCRIPTION
**Pull request by pavanjoshi914**
### Describe the changes you have made in this PR

Extend kind list with few more common kinds used on primal

- `feat`: New feature (non-breaking change which adds functionality)

### Checklist

- [X] Self-review of changed code
- [X] Manual testing
- [X] Added automated tests where applicable
- [X] Update Docs & Guides
- For UI-related changes
- [X] Darkmode
- [X] Responsive layout

### CODE REVIEW

File: src/extension/background-script/actions/nostr/enable.ts [hunk 1]
```diff
@@ -114,13 +114,17 @@ const enable = async (message: MessageAllowanceEnable, sender: Sender) => {
             3, // Update follow list
             4, // Encrypted direct messages
             7, // Reaction
+            6, // Repost
```
Comment (rolznz) [inline src/extension/background-script/actions/nostr/enable.ts:117]: can you put the items in order?

File: src/extension/providers/nostr/types.ts [hunk 2]
```diff
@@ -15,7 +15,25 @@ export enum EventKind {
   Contacts = 3,
   DM = 4,
   Deleted = 5,
+  Repost = 6,
```
Comment (rolznz) [inline src/extension/providers/nostr/types.ts:18]: what are these added types used for and where?

File: src/extension/background-script/actions/nostr/enable.ts [hunk 3]
```diff
@@ -114,13 +114,17 @@ const enable = async (message: MessageAllowanceEnable, sender: Sender) => {
...
             9734, // Zap request
+            10000, // mute list
             10002, // Relay list metadata
+            10003, // bookmarks
```
Comment (rolznz) [inline src/extension/background-script/actions/nostr/enable.ts:121]: Can you also please add `27235` (HTTP Auth)

Comment (rolznz) [reply src/extension/background-script/actions/nostr/enable.ts:121]: oops, sorry - wrong place.

File: src/i18n/locales/en/translation.json [hunk 4]
```diff
@@ -1039,10 +1039,18 @@
...
+          "description": "Blocks spammers and bad actors from your feed"
+        },
         "10002": {
           "title": "relay list",
           "description": "Update preferred relays for discovering your notes and receiving others' notes"
         },
+        "10003": {
+          "title": "bookmarks",
+          "description": "Bookmark user posts"
+        },
```
Comment (rolznz) [inline src/i18n/locales/en/translation.json:1053]: Can you also please add 27235 (HTTP Auth)

Used to upload images in nostrudel

File: src/extension/background-script/actions/nostr/enable.ts (hunk 1 above)
Comment (pavanjoshi914) [reply src/extension/background-script/actions/nostr/enable.ts:117]: resolved

File: src/extension/providers/nostr/types.ts (hunk 2 above)
Comment (pavanjoshi914) [reply src/extension/providers/nostr/types.ts:18]: permission will be set for this kinds in one click while selecting reasonable option in enable screen

Comment (rolznz) [reply src/extension/providers/nostr/types.ts:18]: but in enable.ts above, they are not used (you use numbers again and with comments to describe the number)

**Comment by pavanjoshi914** [comment]
tAck

**Pull request merged**